Vagrant/solrcloudVagrantfile

```diff
@@ -0,0 +1,98 @@
+# Defines our Vagrant environment
+#
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  # Create Zookeeper ensemble server nodes
+  # https://docs.vagrantup.com/v2/vagrantfile/tips.html
+  (1..1).each do |i|
+    config.vm.define "lib-zk#{i}" do |node|
+        node.vm.box = "princeton_box"
+        node.vm.hostname = "lib-zk#{i}"
+        node.vm.network :private_network, ip: "10.0.15.1#{i}"
+        node.vm.network "forwarded_port", guest: 2181, host: "998#{i}"  # zookeeper
+        node.vm.provider "virtualbox" do |vb|
+          vb.memory = "2048"
+        end
+    end
+    config.vm.provision "ansible" do |ansible|
+        ansible.verbose = 'vvv'
+        ansible.playbook = "zookeeper.yml"
+        ansible.limit = "all"
+        ansible.extra_vars = {
+          deploy_user_uid: "1002",
+          lib_zk1_host: "10.0.15.11",
+          lib_zk2_host: "10.0.15.12",
+          lib_zk3_host: "10.0.15.13"
+        }
+        ansible.groups = {
+          "zookeeper" => ["lib-zk#{i}"]
+        }
+    end
+    config.ssh.private_key_path = "pulsys_rsa_key"
+    config.ssh.username = "pulsys"
+  end
+
+  # Create Solr server nodes
+  # https://docs.vagrantup.com/v2/vagrantfile/tips.html
+  (1..1).each do |i|
+    config.vm.define "lib-zk#{i}" do |node|
```

Comment:
This section appears to be a duplicate of the zookeeper section above, except with a different comment.